api/v1beta1/agentserviceconfig_types.go

```diff
@@ -109,6 +109,12 @@ type AgentServiceConfigSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="List of container registries without authentication"
 	// +optional
 	UnauthenticatedRegistries []string `json:"unauthenticatedRegistries,omitempty"`
+	// ImagePullCAConfigMapRef is a reference to a config map containing a certificate authority certificate
```

Comment:
Can you just add a blank line between all the fields and the next field comment?
I see it's inconsistent in this struct, but we can fix that now.